packages/clerk-js/webpack.config.js

```diff
@@ -100,7 +100,7 @@ const svgLoader = () => {
 /** @type { () => (import('webpack').RuleSetRule) } */
 const typescriptLoaderProd = () => {
   return {
-    test: /\.(ts|js)x?$/,
+    test: /\.(js|mjs|jsx|ts|tsx)$/,
```

Comment:
I didn't feel like using `(ts|js|mjs)x?` as that would allow `mjsx` so this is a little bit more verbose but clear

Reply:
you could use `mjs|((ts|js)x?)`

Reply:
Let's aim for clarity here.